nusacrowd/nusa_datasets/su_emot/su_emot.py

```diff
@@ -0,0 +1,131 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from nusacrowd.utils import schemas
+from nusacrowd.utils.configs import NusantaraConfig
+from nusacrowd.utils.constants import DEFAULT_NUSANTARA_VIEW_NAME, DEFAULT_SOURCE_VIEW_NAME, Tasks
+
+_DATASETNAME = "su_emot"
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_NUSANTARA_VIEW_NAME
+
+_LANGUAGES = ["sun"]
+_LOCAL = False
+_CITATION = """\
+@INPROCEEDINGS{
+9297929,  
+author={Putra, Oddy Virgantara and Wasmanson, Fathin Muhammad and Harmini, Triana and Utama, Shoffin Nahwa},  
+booktitle={2020 International Conference on Computer Engineering, Network, and Intelligent Multimedia (CENIM)},   
+title={Sundanese Twitter Dataset for Emotion Classification},   
+year={2020},  
+volume={},  
+number={},  
+pages={391--395},  
+doi={10.1109/CENIM51130.2020.9297929}
+}
+"""
+
+_DESCRIPTION = """\
+This is a dataset for emotion classification of Sundanese text. The dataset is gathered from Twitter API between January and March 2019 with 2518 tweets in total. 
+The tweets filtered by using some hashtags which are represented Sundanese emotion, for instance, #persib, #corona, #saredih, #nyakakak, #garoblog, #sangsara, #gumujeng, #bungah, #sararieun, #ceurik, and #hariwang. 
+This dataset contains four distinctive emotions: anger, joy, fear, and sadness. Each tweet is annotated using related emotion. For data
+validation, the authors consulted a Sundanese language teacher for expert validation.
+"""
+_HOMEPAGE = "https://github.com/virgantara/sundanese-twitter-dataset"
+
+_LICENSE = "UNKNOWN"
+
+_URLS = {
+	"datasets": "https://raw.githubusercontent.com/virgantara/sundanese-twitter-dataset/master/newdataset.csv"
+}
+
+_SUPPORTED_TASKS = [Tasks.EMOTION_CLASSIFICATION]
+
+
+_SOURCE_VERSION = "1.0.0"
+
+_NUSANTARA_VERSION = "1.0.0"
+
+
+class suEmot(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class SuEmot(datasets.GeneratorBasedBuilder):
```

Would be better if this class name is camel case with capitalzed front letter